### ISSUE TITLE
[CTW/40v40] Welcome to Wool Circle

### ISSUE THREAD

**Issue by Noorthside** [OvercastCommunity/CommunityMaps#257]
### Checklist
Check what applies to you. *Add an X between the brackets or click the checkboxes when you have submitted the issue.*
- [x] I have [pruned](https://pgm.dev/docs/guides/packaging/pruning-chunks) the map
- [x] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the README
- [x] I have created an XML file
- [x] I have created an map image
- [x] I have uploaded the map zip file to a file sharing service
- [x] The map has been tested locally to make sure it works

# Map Name
Welcome to Wool Circle

## Gamemode & Map Description  
CTW, 3 Wools and inspired by Welcome to Wool Square.

## Team Sizes
2 Teams, 40 vs 40

## Screenshots
https://imgur.com/a/UTjjZ2e

# Map Image
![welcome to wool circle png](https://user-images.githubusercontent.com/65872499/129277286-9e9d4d46-efb0-4665-89a6-66d6d8896b4b.png)

**Comment by CoWinkKeyDinkInc** [comment]
You forgot to add a link to the map download and XML

**Comment by CoWinkKeyDinkInc** [comment]
Updated in 8c78d7f888cd6bcf75d785e8046e2e4f7b7bbf71

**Comment by CoWinkKeyDinkInc** [comment]
It should be more clear that the water is the portals at spawn. There's no height limit so that should be added. Also you should utilize more of the empty space that is between the circular lanes to the wool room

**Comment by calcastor** [comment]
I don't think this will work super well in regular pools, but I went ahead and made some changes including dropping it back to 40v40 and some other minor stuff like permitting wool room building (though making sure you can't mine out the floor of the two side wool rooms by changing the wood to bedrock). I've removed the dev phase on it so it can be sponsored. Thank you for submitting!
